matrix-sdk/src/main/java/org/matrix/androidsdk/crypto/MXCrypto.java

```diff
@@ -825,21 +825,6 @@ public void setDeviceVerification(final int verificationStatus, final String dev
             return;
         }
 
-        final List<String> userRoomIds = new ArrayList<>();
-
-        Collection<Room> rooms = mSession.getDataHandler().getStore().getRooms();
-
-        for (Room room : rooms) {
-            if (room.isEncrypted()) {
-                RoomMember roomMember = room.getMember(userId);
-
-                // test if the user joins the room
-                if ((null != roomMember) && TextUtils.equals(roomMember.membership, RoomMember.MEMBERSHIP_JOIN)) {
-                    userRoomIds.add(room.getRoomId());
-                }
-            }
-        }
```

Comment:
Ok to remove this dead code. Missing cleanup from 444644570ac6b424b700d38879a839247ba80f4f